modules/examiner/php/examiner.class.inc

```diff
@@ -336,6 +362,7 @@ class Examiner extends \NDB_Menu_Filter_Form
             $deps,
             array(
              $baseURL . "/examiner/js/columnFormatter.js",
```

Comment:
No need for this once `columnFormatter` is included as a function within the React Component.